app/controllers/catalog_controller.rb

```diff
@@ -764,11 +767,14 @@ def ot_remove_submit
   def ot_add
     assert_privileges("orchestration_template_add")
     ot_type = x_node == "root" ? "OrchestrationTemplateCfn" : node_name_to_template_name(x_node)
-    @edit = {:new => {:name        => "",
-                      :description => "",
-                      :content     => "",
-                      :type        => ot_type,
-                      :draft       => false}}
+    @edit = {:new => {:name          => "",
+                      :description   => "",
+                      :content       => "",
+                      :type          => ot_type,
+                      :draft         => false,
+                      :manager_id    => ''}}
+    @edit[:new][:available_managers] =
+      ManageIQ::Providers::Openstack::CloudManager.all.collect { |t| [t.name, t.id] }.sort || []
```

Comment:
there is `OrchestrationTemplateCfn.eligible_managers`
does this method only deal with Cfn templates?